tests/end-to-end/api/24-methods.js

```diff
@@ -1352,6 +1352,29 @@ describe('Meteor.methods', function() {
 				})
 				.end(done);
 		});
+
+		it('should not parse urls with <> in the sent message', (done) => {
```

Comment:
Can you add more tests? For scenarios like:
- Sending a message in a thread
- Editing a message and adding a new url with <>
- Edit a message to put a url into <>
- Edit a message to remove the <> from a url
- Sending URLs with a lot of query params to see if the regex supports it.

Reply:
Okay!


I will add the above tests under a new `describe` block. 🧪

Reply:
![image](https://user-images.githubusercontent.com/55396651/116301525-c161c600-a7bd-11eb-9e71-d67da3ad7650.png)

Added the tests!